nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -67,22 +67,38 @@ def __init__(self, cfg: DictConfig, trainer=None):
         vocabulary = self.tokenizer.tokenizer.get_vocab()
 
         # Set the new vocabulary
-        cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
+        if "params" in cfg.decoder:
+            cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
+        else:
+            cfg.decoder.vocabulary = ListConfig(list(vocabulary.values()))
 
         # Override number of classes if placeholder provided
-        if cfg.decoder.params['num_classes'] < 1:
+        if "params" in cfg.decoder:
+            num_c = cfg.decoder["params"]["num_classes"]
```

Comment:
Rename this variable to `num_classes` for clarity.

Reply:
done